scripts/controllers/aboutController.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+(function(module) {
+  var aboutController = {};
+
+  aboutController.show = function () {
+    $('#Home').fadeIn(1000)
```

Comment:
Don't forget to `fadeOut` all the things you don't want to see. It's good to just remove everything and then explicitly show what you want.